src/ui/profile_preferences.ui

```diff
@@ -184,13 +181,8 @@
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_guiLanguage">
             <property name="toolTip">
-             <string>&lt;p&gt;Can you help translate Mudlet? &lt;span style=&quot;font-weight:600;&quot;&gt;https://www.mudlet.org/translate&lt;/span&gt;&lt;/p&gt;</string>
+             <string>&lt;p&gt;Can you help translate Mudlet?&lt;/p&gt;&lt;p&gt;If so, please visit: &lt;b&gt;https://www.mudlet.org/translate&lt;/b&gt;.&lt;/p&gt;</string>
```

Comment:
I don't see the improvement here for the user, and it just adds a bit of work to translators.

Reply:
Existing:
![Screenshot_20190802_223908](https://user-images.githubusercontent.com/6163092/62400226-65312a00-b576-11e9-8f7b-9cf5e435109c.png)

After this PR:
![Screenshot_20190802_223340](https://user-images.githubusercontent.com/6163092/62400259-82fe8f00-b576-11e9-8f9b-88ff1b43aa35.png)

Better layout (in two paragraphs) and a bit more *friendly* IMVHO... what do other @Mudlet/ui-review people think?